packages/svelte2tsx/src/htmlxtojsx/index.ts

```diff
@@ -1,12 +1,11 @@
 import { Node } from 'estree-walker';
 import MagicString from 'magic-string';
-import svelte from 'svelte/compiler';
+import * as svelte from 'svelte/compiler';
```

Comment:
I think these import changes can be reverted because "esModuleInterop" should take care of it.

Reply:
nope 
<img width="764" alt="Code_2021-02-25_07-47-53" src="https://user-images.githubusercontent.com/30108880/109114374-d2e70d00-773d-11eb-8047-ebfe85efc6b5.png">

Reply:
I stand corrected. After some digging I think this is actually correct behavior when not using Rollup's interop magic, since the compiler has no default export. What could be changed though is to use a named import here and in the other location instead of `* as svelte`, because in both instances only parse/walk is used. If you could do that and resolve the merge conflict then I'm good to merge this.